shimmed-buildpacks/gradle/buildpack.toml

```diff
@@ -20,6 +20,9 @@ id = "heroku-20"
 [[stacks]]
 id = "io.buildpacks.stacks.bionic"
```

Comment:
Should we remove the other stacks, now we've added the wildcard stack?
Whilst the spec supports having both, it feels redundant to do so.

Reply:
`heroku/maven` also has both right now. Given that both will get rewritten very soon - I'd rather trade skipping another CI run on this PR since it takes ages to complete for a change with no direct impact.